### PR TITLE
feat: add myacls view

### PR DESCRIPTION
**Pull request by open-dynaMIX**
- Auto creation of related acl object make testing harder
 - MyACLViewSet: This permits a user to fetch its own ACLs

### CODE REVIEW

File: emeis/core/urls.py [hunk 1]
```diff
@@ -12,7 +12,13 @@
 r.register(r"acls", views.ACLViewSet)
 
 urlpatterns = [
-    url(r"^me", views.MeViewSet.as_view({"get": "retrieve"}), name="me-detail")
+    url(r"^me", views.MeViewSet.as_view({"get": "retrieve"}), name="me-detail"),
+    url(
+        r"^myacls/(?P<pk>[^/.]+)$",
```
Comment (sbor23) [inline emeis/core/urls.py:17]: More of a general input, but we should consider moving from `url` to `path` for new implementations.
https://docs.djangoproject.com/en/2.2/ref/urls/#path

It's nicer and most of all `url` docs state that "It’s likely to be deprecated in a future release."

File: emeis/core/views.py [hunk 2]
```diff
@@ -14,6 +16,23 @@ def get_object(self, *args, **kwargs):
...
+    """MyACL view returns current users ACLs."""
+
+    queryset = models.ACL.objects.all()
+    serializer_class = serializers.MyACLSerializer
+
+    def get_object(self):
+        pk = self.kwargs.pop("pk")
+        acl = get_object_or_404(models.ACL, pk=pk)
+        if acl.user != self.request.user:
+            raise PermissionDenied()
```
Comment (sbor23) [inline emeis/core/views.py:29]: Would it make sense to handle this in permissions? The name of the exception is kinda leading there...

File: emeis/core/urls.py (hunk 1 above)
Comment (winged) [reply emeis/core/urls.py:17]: :+1:

File: emeis/core/views.py (hunk 2 above)
Comment (winged) [reply emeis/core/views.py:29]: Not in this view: Here it's the express purpose that you're only ever see your own ACLs.

File: emeis/core/urls.py (hunk 1 above)
Comment (open-dynaMIX) [reply emeis/core/urls.py:17]: Absolutely! Done.

**Pull request merged**